pubsub/pubsub_test.go

```diff
@@ -6,11 +6,9 @@ import (
 	"github.com/stvp/assert"
 )
 
-type MessageStructTest struct{}
-
 func TestPublish(t *testing.T) {
 	key := "TestPublish"
-	data := MessageStructTest{}
```

Comment:
@antho1404 @krhubert I think both version of this line is error prone. We should use a struct type with a field and set a value to it to do a proper assertion on struct equality.

_"Struct values are comparable if all their fields are comparable. Two struct values are equal if their corresponding non-blank fields are equal."_

Reply:
👍

Reply:
Done in https://github.com/mesg-foundation/core/pull/321/files